src/utils/isPlainObject.js

```diff
@@ -0,0 +1,3 @@
+export default function isPlainObject(obj) {
+  return obj ? typeof obj === 'object' && Object.getPrototypeOf(obj) === Object.prototype : false;
```

Comment:
@ooflorent Are we trying to avoid the object above or `Object.create(null)`? It is perfectly fine for someone to have a React state that is not a plain object. ImmutableJS is used for state and it is not a plain object.